contracts/WildcardSteward_v2.sol

```diff
@@ -291,6 +314,20 @@ contract WildcardSteward_v2 is Initializable {
         admin = _admin;
     }
 
+    function changeWithdrawCheckerAdmin(address _withdrawCheckerAdmin)
+        public
+        onlyAdmin
+    {
+        withdrawCheckerAdmin = _withdrawCheckerAdmin;
+    }
+
+    function setGlobal(uint256 _globalBenefactorDailyWithdrawalLimit)
+        external
+        onlyAdmin
+    {
```

Comment:
Think a basic require like < 10eth or something may be good just to hard cap this?

Reply:
Yeah, makes sense. Would be good to add that